scripts/tests/test-ci-all.sh

```diff
@@ -215,35 +215,42 @@ done
 
 parsed_test_commands=$(printf "%s\n" "${tests_with_versions[@]}")
 
-delay="${FM_TEST_CI_ALL_DELAY:-$((64 / $(nproc) + 1))}"
-load="${FM_TEST_CI_ALL_MAX_LOAD:-1000}"
 parallel_args=()
-export parallel_jobs='0'
+export parallel_jobs='1'
 
 if [ -z "${CI:-}" ]; then
   parallel_args+=(--eta)
 fi
 
+if [ -n "${FM_TEST_CI_ALL_JOBS:-}" ]; then
+  # when specifically set, use the env var
+  parallel_args+=(--jobs "${FM_TEST_CI_ALL_JOBS}")
+elif [ -n "${CI:-}" ]; then
+  # in CI, we know number of cpus works OK
+  parallel_args+=(--jobs +0)
```

Comment:
Did you want to keep this `0` instead of `+0` so we can run as many jobs as possible in CI, or did you specifically want to switch back to jobs == num_cpu?

Reply:
In CI, let's do jobs == num_cpu for now too.